dev/generator/main.go

```diff
@@ -221,25 +220,26 @@ func buildPackage(packagesBasePath string, p util.Package) error {
 		}
 
 		tarGzName := p.Name + "-" + p.Version
-		copiedPackagePath := filepath.Join(tarGzDirPath, tarGzName)
+		dst := filepath.Join(tarGzDirPath, tarGzName)
 
 		// As the package directories are now {packagename}/{version} when just running tar, the dir inside
 		// the package had the wrong name. Using `-s` or `--transform` for some reason worked on the command line
 		// but not when run through Golang. So the hack for now is to just copy over all files with the correct name
 		// and then run tar on it.
 		// This could become even useful in the future as things like images or videos should potentially not be part of
 		// a tar.gz to keep it small.
-		err := CopyPackage(packagesBasePath+"/"+p.Name+"/"+p.Version+"/", copiedPackagePath)
+		src := packagesBasePath + "/" + p.Name + "/" + p.Version + "/"
```

Comment:
```suggestion
		src := filepath.Join(packagesBasePath, p.Name, p.Version) + "/"
```

I think we have to leave in the last `/`?